conda_smithy/lint_recipe.py

```diff
@@ -86,7 +88,13 @@ def lintify(meta, recipe_dir=None):
     if 'unknown' == license.strip():
         lints.append('The recipe license cannot be unknown.')
 
-    # 6: Selectors should be in a tidy form.
+    # 6: License family must be valid (conda-build checks for that)
```

Comment:
Let's go ahead and put it at the end. May need to rebase though as I just merged PR ( https://github.com/conda-forge/conda-smithy/pull/334 ).